src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int aside;
+    private int bside;
+    private int heigth;
+
+    public IsoscelesTrapezoid(int aside, int bside, int heigth, String color) {
+        this.aside = aside;
+        this.bside = bside;
+        this.heigth = heigth;
+        setColor(color);
+        setFigureType("isosceles trapezoid");
+        calculateArea();
+    }
```

Comment:
See my comments to the `Figure` class, firstly change them, and then you will need to change all the inheritors.